channels/us.m3u

```diff
@@ -530,13 +530,13 @@ https://bloomberg-bloombergtv-1-es.samsung.wurl.com/manifest/playlist.m3u8
 https://liveprodusphoenixeast.akamaized.net/USPhx-HD/Channel-TX-USPhx-AWS-virginia-1/Source-USPhx-16k-1-s6lk2-BP-07-02-81ykIWnsMsg_live.m3u8
 #EXTINF:-1 tvg-id="BloombergUSDesktop.us" tvg-name="Bloomberg US Desktop" tvg-country="US" tvg-language="English" tvg-logo="http://cdn-profiles.tunein.com/s47135/images/logog.png" group-title="Business",Bloomberg US Desktop
 https://liveprodusphoenixeast.global.ssl.fastly.net/USPhx-HD/Channel-TX-USPhx-AWS-virginia-1/Source-USPhx-16k-1-s6lk2-BP-07-03-0Yn1cQZHOtP_live.m3u8
-#EXTINF:-1 tvg-id="BloombergMeiGuo.us" tvg-name="Bloomberg(美国)" tvg-country="US" tvg-language="English" tvg-logo="http://cdn-profiles.tunein.com/s47135/images/logog.png" group-title="Business",Bloomberg(美国)
+#EXTINF:-1 tvg-id="BloombergMeiGuo.us" tvg-name="Bloomberg (美国)" tvg-country="US" tvg-language="English" tvg-logo="http://cdn-profiles.tunein.com/s47135/images/logog.png" group-title="Business",Bloomberg (美国)
```

Comment:
Not chinese version or source, chinese part should be removed entirely

Reply:
Okay, I have seen that in a few others